### DISCUSSION TITLE
All in i need to know about github

### DISCUSSION THREAD

**Discussion by ogbeubah-star** [community/community#120097] [New to GitHub]
I don't have full knowledge of Git Hub. How can git hub help my career as a developer

**Comment by kalebers** [comment]
Hello!

Other fellow developers can join me on this haha But in summary, using GitHub to store your projects and help the open source community shows that you are interest and have knowledge while building portfolio. Many companies look around the internet to see more of your work, so it's good to have a good profile with your main projects and repos for study. 

And for the Open source community, I would say that be active in the Open Source projects  helps you to work with another developers, think together to improve or create a tool/software and pushes you to create your developer side, improving your way of thinking and problem solving. You gain experience as a developer and help the community grow as well the tool or software you are helping to develop and solve issues. 

I'll leave some links that may be interesting to read, and don't hesitate to interact with the community and show your work on your profile, we all are always trying to improve ourselves and help other developers :) 

- https://www.codecademy.com/resources/blog/how-to-use-github-to-strengthen-your-resume/#:~:text=Developers%20use%20GitHub%20to%20manage,there%20to%20work%20on%20later.
- https://www.linkedin.com/advice/0/youre-looking-advance-your-career-web-applications-6if3c
- https://ritikchourasiya.medium.com/why-you-need-a-github-profile-as-a-software-developer-483a1bf02d17

**Comment by ghost** [comment]
Hi, Git is a distributed version control system that allows developers to track changes in their codebase.
GitHub, on the other hand, is a web-based platform built around Git that enhances collaboration and also project management. It serves as a central hub for hosting Git repositories.
basically Git and GitHub serve as indispensable tools to developers, its a platform to manage code effectively, collaborate with peers and showcase skills
